packages/cli/src/lingui-compile.ts

```diff
@@ -180,41 +192,35 @@ if (require.main === module) {
     })
     .parse(process.argv)
 
-  const config = getConfig({ configPath: program.config })
+  const options = program.opts<CliOptions>()
```

Comment:
this is biggest difference in versions, 

https://github.com/tj/commander.js/blob/master/CHANGELOG.md#migration-tips-1